components/AppLayout/AppHeader.js

```diff
@@ -1,66 +1,244 @@
+    backgroundColor: theme.palette.secondary[50],
+    [theme.breakpoints.up('md')]: {
+      backgroundColor: 'inherit',
+      height: 'auto',
+      width: 'auto',
+      fontSize: 20,
+      padding: '0 10px',
+    },
+  },
+  tab: {
+    padding: 8,
+    flex: '1',
+    lineHeight: `calc(${TABS_HEIGHT}px - 8px * 2)`,
+    textAlign: 'center',
+    textDecoration: 'none',
+    whiteSpace: 'nowrap',
+    fontWeight: 500,
+    letterSpacing: 0.75,
+    color: theme.palette.secondary[300],
+    [theme.breakpoints.up('md')]: {
+      color: theme.palette.secondary[500],
+      padding: '0 10px',
+    },
+  },
+  activeTab: {
+    color: theme.palette.secondary[500],
+    [theme.breakpoints.up('md')]: {
+      color: theme.palette.primary.main,
+    },
+  },
+  menuToggleButton: {
+    width: MENU_BUTTON_WIDTH,
+    height: TABS_HEIGHT,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    background: theme.palette.secondary[300],
+    color: theme.palette.common.white,
+  },
+  profileMenu: {
+    marginTop: 50,
+    backgroundColor: theme.palette.secondary.main,
+    color: theme.palette.common.white,
+    overflow: 'inherit',
+  },
+  divider: {
+    backgroundColor: theme.palette.secondary[400],
+  },
+  listIcon: {
+    color: theme.palette.secondary[300],
+    minWidth: 0,
+    paddingRight: 8,
+  },
+  login: {
+    borderRadius: 70,
+    border: `1px solid ${theme.palette.secondary[500]}`,
+  },
+}));
+
+const Links = ({ classes }) => (
+  <div className={classes.tabs}>
+    <NavLink
+      href="/articles"
+      className={classes.tab}
+      activeClassName={classes.activeTab}
+    >
+      {t`Collected Messages`}
+    </NavLink>
+    <NavLink
+      href="/replies"
+      className={classes.tab}
+      activeClassName={classes.activeTab}
+    >
+      {t`Replies`}
+    </NavLink>
+    <NavLink
+      external
+      href={EDITOR_FACEBOOK_GROUP}
+      className={cx(classes.tab, 'hidden-xs')}
+    >{t`Editor forum`}</NavLink>
+    <NavLink
+      external
+      href={PROJECT_HACKFOLDR}
+      className={classes.tab}
+    >{t`About`}</NavLink>
+  </div>
+);
 
-        @media screen and (min-width: 768px) {
-          .root {
-            padding: 0 40px;
-          }
-        }
-      `}</style>
+function AppHeader({ onMenuButtonClick, user, openLoginModal, logout }) {
+  const [anchor, setAnchor] = useState(false);
+  const classes = useStyles();
+  const theme = useTheme();
+
+  const openProfileMenu = useCallback(e => setAnchor(e.currentTarget), [
+    anchor,
+  ]);
+  const closeProfileMenu = useCallback(() => setAnchor(null), [anchor]);
+
+  return (
+    <header className={classes.root}>
+      <div className={classes.top}>
+        <div className={classes.flex}>
+          <a href="/">
+            <img
+              className={classes.logo}
+              src={mobileLogo}
+              srcSet={`${desktopLogo} ${theme.breakpoints.values.md}w`}
+              alt=""
```

Comment:
It's showing desktop logo on mobile screen:
![image](https://user-images.githubusercontent.com/108608/78016371-4a20ef00-737d-11ea-97d6-e334803519dd.png)

I think its because the lack of `sizes` property documented here: https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#Resolution_switching_Different_sizes

Another thing is, the logo looks blurry. I think using SVGs instead of PNGs should fix that.

Reply:
not sure why it looks so blur in original size, but it looks decent when zoomed in, really weired
![截圖 2020-03-31 下午8 35 57](https://user-images.githubusercontent.com/12843409/78027325-c3c1d880-738f-11ea-952f-1bef8cdd6a9a.png)

Reply:
also the bug of misplacing image was fixed, but I use the picture method mentioned here instead: 
https://cythilya.github.io/2018/08/24/responsive-images/